scenes/quests/lore_quests/quest_001/1_music_puzzle/components/xylophone/musical_rock.gd

```diff
@@ -22,12 +23,13 @@ func _ready() -> void:
 
 
 func _on_interaction_started(_player: Player, _from_right: bool) -> void:
+	note_played.emit()
```

Comment:
:+1: I remember we talked about doing this instead of having a "is demo" flag. Better.